app/docs/0.11.x/loadbalancing.md

```diff
@@ -182,6 +182,112 @@ __Exception__: When a DNS record has `ttl=0`, the hostname will be added
 as a single target, with the specified weight. Upon every proxied request
 to this target it will query the nameserver again.
 
+### **Blue-Green Deployments**
+
+Using the ring-balancer a [blue-green deployment][blue-green-canary] can be easily orchestrated for 
```

Comment:
This link (`[blue-green-canary]`) is missing from the markdown (it is at the bottom of the 0.10 version of this page).